styles/less/search.less

```diff
@@ -45,9 +45,6 @@
 	.eslint-navbar {
 		clear: both;
 	}
-	.algolia-autocomplete {
```

Comment:
I don't see any references to this class in our templates nor when I search the DOM. I believe this is no longer used.